post_processing/pylbo/data_containers.py

```diff
@@ -586,7 +586,9 @@ def get_derived_eigenfunctions(self, ev_guesses=None, ev_idxs=None) -> np.ndarra
             getter_func=self.filereader.read_derived_eigenfunction,
         )
 
-    def get_nearest_eigenvalues(self, ev_guesses) -> tuple(np.ndarray, np.ndarray):
+    def get_nearest_eigenvalues(
+        self, ev_guesses, min_distance=0.0
+    ) -> tuple(np.ndarray, np.ndarray):
```

Comment:
I needed this functionality for tracking of sequences of eigenvalues: which eigenvalue is closest to the current one in the sequence, but distinct from it. Providing a minimum distance to the ev_guesses makes that work. If this doesn't need to be standard functionality, I could keep it on a separate local branch, and otherwise a dedicated function indeed might be a good idea.

Reply:
Sure, if you put it in a method `get_eigenvalues_at_distance()` or something like that we can easily distinguish between the two (and test them separately).

Reply:
I now let `get_nearest_eigenvalues()` refer to `get_eigenvalues_at_distance()` since the latter is just a more general version of the former. Let me know if your agree with this.

Reply:
perfect!